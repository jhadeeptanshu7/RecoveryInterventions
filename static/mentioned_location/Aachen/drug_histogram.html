



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="7850c6fb-2028-4de0-9a54-dd15e98b4212"></div>
            
          
        
      
      
        <script type="application/json" id="349099">
          {"70fd2b8d-3363-4f6d-8a3b-9c6faab6959f":{"roots":{"references":[{"attributes":{"callback":null,"factors":["speed","black","north","fire","h","lines","rocket","nexus","crown","shrooms","strain","alkaloids","kratom","acid","sub","snow","m1","white","opiates","25i","methadone","coke","molly","mph","weed","glass","heroin","hits","e","oxy","k","l","strains","ms"],"range_padding":0.01},"id":"348998","type":"FactorRange"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"349032","type":"VBar"},{"attributes":{"plot":null,"text":"Drug Counts"},"id":"348995","type":"Title"},{"attributes":{"data_source":{"id":"349030","type":"ColumnDataSource"},"glyph":{"id":"349031","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"349032","type":"VBar"},"selection_glyph":null,"view":{"id":"349034","type":"CDSView"}},"id":"349033","type":"GlyphRenderer"},{"attributes":{"below":[{"id":"349006","type":"CategoricalAxis"}],"left":[{"id":"349010","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"349006","type":"CategoricalAxis"},{"id":"349009","type":"Grid"},{"id":"349010","type":"LinearAxis"},{"id":"349014","type":"Grid"},{"id":"349023","type":"BoxAnnotation"},{"id":"349033","type":"GlyphRenderer"}],"title":{"id":"348995","type":"Title"},"toolbar":{"id":"349021","type":"Toolbar"},"x_range":{"id":"348998","type":"FactorRange"},"x_scale":{"id":"349002","type":"CategoricalScale"},"y_range":{"id":"349000","type":"DataRange1d"},"y_scale":{"id":"349004","type":"LinearScale"}},"id":"348996","subtype":"Figure","type":"Plot"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"349023","type":"BoxAnnotation"},{"attributes":{},"id":"349020","type":"HelpTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"349015","type":"PanTool"},{"id":"349016","type":"WheelZoomTool"},{"id":"349017","type":"BoxZoomTool"},{"id":"349018","type":"SaveTool"},{"id":"349019","type":"ResetTool"},{"id":"349020","type":"HelpTool"}]},"id":"349021","type":"Toolbar"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"348996","subtype":"Figure","type":"Plot"},"ticker":{"id":"349007","type":"CategoricalTicker"}},"id":"349009","type":"Grid"},{"attributes":{},"id":"349004","type":"LinearScale"},{"attributes":{"axis_label":"Drug","formatter":{"id":"349037","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"348996","subtype":"Figure","type":"Plot"},"ticker":{"id":"349007","type":"CategoricalTicker"}},"id":"349006","type":"CategoricalAxis"},{"attributes":{},"id":"349007","type":"CategoricalTicker"},{"attributes":{"callback":null,"start":0},"id":"349000","type":"DataRange1d"},{"attributes":{},"id":"349002","type":"CategoricalScale"},{"attributes":{},"id":"349040","type":"Selection"},{"attributes":{},"id":"349041","type":"UnionRenderers"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"349031","type":"VBar"},{"attributes":{"callback":null,"data":{"top":[2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["speed","black","north","fire","h","lines","rocket","nexus","crown","shrooms","strain","alkaloids","kratom","acid","sub","snow","m1","white","opiates","25i","methadone","coke","molly","mph","weed","glass","heroin","hits","e","oxy","k","l","strains","ms"]},"selected":{"id":"349040","type":"Selection"},"selection_policy":{"id":"349041","type":"UnionRenderers"}},"id":"349030","type":"ColumnDataSource"},{"attributes":{},"id":"349019","type":"ResetTool"},{"attributes":{},"id":"349018","type":"SaveTool"},{"attributes":{"source":{"id":"349030","type":"ColumnDataSource"}},"id":"349034","type":"CDSView"},{"attributes":{},"id":"349037","type":"CategoricalTickFormatter"},{"attributes":{},"id":"349039","type":"BasicTickFormatter"},{"attributes":{},"id":"349011","type":"BasicTicker"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"349039","type":"BasicTickFormatter"},"plot":{"id":"348996","subtype":"Figure","type":"Plot"},"ticker":{"id":"349011","type":"BasicTicker"}},"id":"349010","type":"LinearAxis"},{"attributes":{"overlay":{"id":"349023","type":"BoxAnnotation"}},"id":"349017","type":"BoxZoomTool"},{"attributes":{},"id":"349016","type":"WheelZoomTool"},{"attributes":{},"id":"349015","type":"PanTool"},{"attributes":{"dimension":1,"plot":{"id":"348996","subtype":"Figure","type":"Plot"},"ticker":{"id":"349011","type":"BasicTicker"}},"id":"349014","type":"Grid"}],"root_ids":["348996"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('349099').textContent;
                  var render_items = [{"docid":"70fd2b8d-3363-4f6d-8a3b-9c6faab6959f","roots":{"348996":"7850c6fb-2028-4de0-9a54-dd15e98b4212"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>