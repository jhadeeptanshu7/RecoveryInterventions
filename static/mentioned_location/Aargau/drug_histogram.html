



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="6c3ad29f-2370-450e-8d73-f861f14876f0"></div>
            
          
        
      
      
        <script type="application/json" id="572885">
          {"1d5f71dc-2970-406b-8edb-3b9859c84f78":{"roots":{"references":[{"attributes":{"overlay":{"id":"572809","type":"BoxAnnotation"}},"id":"572803","type":"BoxZoomTool"},{"attributes":{},"id":"572826","type":"Selection"},{"attributes":{},"id":"572827","type":"UnionRenderers"},{"attributes":{},"id":"572790","type":"LinearScale"},{"attributes":{"source":{"id":"572816","type":"ColumnDataSource"}},"id":"572820","type":"CDSView"},{"attributes":{"axis_label":"Drug","formatter":{"id":"572823","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"572782","subtype":"Figure","type":"Plot"},"ticker":{"id":"572793","type":"CategoricalTicker"}},"id":"572792","type":"CategoricalAxis"},{"attributes":{},"id":"572793","type":"CategoricalTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"572782","subtype":"Figure","type":"Plot"},"ticker":{"id":"572793","type":"CategoricalTicker"}},"id":"572795","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"572825","type":"BasicTickFormatter"},"plot":{"id":"572782","subtype":"Figure","type":"Plot"},"ticker":{"id":"572797","type":"BasicTicker"}},"id":"572796","type":"LinearAxis"},{"attributes":{},"id":"572797","type":"BasicTicker"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"572817","type":"VBar"},{"attributes":{"callback":null,"data":{"top":[2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["serotonin","mdma","clonidine","nicotine","cbd","benzo","ssri","dxm","gt","dope","dopamine","diazepam","gabaergics","ritalin","sub","sodium","u4","phenibut","salvia","black","vicodin","amitriptyline","pregabalin","baclofen","mary","choline","oil","north","coke","dph","weed","norepinephrine","lunesta","amphetamine","valium","e","h","l","piracetam","benzos","adderall","fasoracetam","opiates","gaba"]},"selected":{"id":"572826","type":"Selection"},"selection_policy":{"id":"572827","type":"UnionRenderers"}},"id":"572816","type":"ColumnDataSource"},{"attributes":{},"id":"572805","type":"ResetTool"},{"attributes":{"data_source":{"id":"572816","type":"ColumnDataSource"},"glyph":{"id":"572817","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"572818","type":"VBar"},"selection_glyph":null,"view":{"id":"572820","type":"CDSView"}},"id":"572819","type":"GlyphRenderer"},{"attributes":{},"id":"572788","type":"CategoricalScale"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"572818","type":"VBar"},{"attributes":{"callback":null,"start":0},"id":"572786","type":"DataRange1d"},{"attributes":{"dimension":1,"plot":{"id":"572782","subtype":"Figure","type":"Plot"},"ticker":{"id":"572797","type":"BasicTicker"}},"id":"572800","type":"Grid"},{"attributes":{},"id":"572801","type":"PanTool"},{"attributes":{},"id":"572802","type":"WheelZoomTool"},{"attributes":{},"id":"572825","type":"BasicTickFormatter"},{"attributes":{},"id":"572804","type":"SaveTool"},{"attributes":{},"id":"572823","type":"CategoricalTickFormatter"},{"attributes":{},"id":"572806","type":"HelpTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"572801","type":"PanTool"},{"id":"572802","type":"WheelZoomTool"},{"id":"572803","type":"BoxZoomTool"},{"id":"572804","type":"SaveTool"},{"id":"572805","type":"ResetTool"},{"id":"572806","type":"HelpTool"}]},"id":"572807","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"572809","type":"BoxAnnotation"},{"attributes":{"callback":null,"factors":["serotonin","mdma","clonidine","nicotine","cbd","benzo","ssri","dxm","gt","dope","dopamine","diazepam","gabaergics","ritalin","sub","sodium","u4","phenibut","salvia","black","vicodin","amitriptyline","pregabalin","baclofen","mary","choline","oil","north","coke","dph","weed","norepinephrine","lunesta","amphetamine","valium","e","h","l","piracetam","benzos","adderall","fasoracetam","opiates","gaba"],"range_padding":0.01},"id":"572784","type":"FactorRange"},{"attributes":{"below":[{"id":"572792","type":"CategoricalAxis"}],"left":[{"id":"572796","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"572792","type":"CategoricalAxis"},{"id":"572795","type":"Grid"},{"id":"572796","type":"LinearAxis"},{"id":"572800","type":"Grid"},{"id":"572809","type":"BoxAnnotation"},{"id":"572819","type":"GlyphRenderer"}],"title":{"id":"572781","type":"Title"},"toolbar":{"id":"572807","type":"Toolbar"},"x_range":{"id":"572784","type":"FactorRange"},"x_scale":{"id":"572788","type":"CategoricalScale"},"y_range":{"id":"572786","type":"DataRange1d"},"y_scale":{"id":"572790","type":"LinearScale"}},"id":"572782","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Drug Counts"},"id":"572781","type":"Title"}],"root_ids":["572782"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('572885').textContent;
                  var render_items = [{"docid":"1d5f71dc-2970-406b-8edb-3b9859c84f78","roots":{"572782":"6c3ad29f-2370-450e-8d73-f861f14876f0"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>