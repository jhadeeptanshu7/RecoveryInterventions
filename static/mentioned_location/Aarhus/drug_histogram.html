



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="be2c1604-b3df-4edf-aee0-755e3ccb2788"></div>
            
          
        
      
      
        <script type="application/json" id="357425">
          {"5b47b444-ddae-40e3-9b26-3532ef1dc2a5":{"roots":{"references":[{"attributes":{},"id":"357365","type":"BasicTickFormatter"},{"attributes":{},"id":"357345","type":"ResetTool"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.09},"x":{"field":"x"}},"id":"357357","type":"VBar"},{"attributes":{"callback":null,"data":{"top":[1,1,1,1],"x":["strain","joints","fish","fluoxetine"]},"selected":{"id":"357366","type":"Selection"},"selection_policy":{"id":"357367","type":"UnionRenderers"}},"id":"357356","type":"ColumnDataSource"},{"attributes":{},"id":"357333","type":"CategoricalTicker"},{"attributes":{"axis_label":"Drug","formatter":{"id":"357363","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"357322","subtype":"Figure","type":"Plot"},"ticker":{"id":"357333","type":"CategoricalTicker"}},"id":"357332","type":"CategoricalAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"357322","subtype":"Figure","type":"Plot"},"ticker":{"id":"357333","type":"CategoricalTicker"}},"id":"357335","type":"Grid"},{"attributes":{"source":{"id":"357356","type":"ColumnDataSource"}},"id":"357360","type":"CDSView"},{"attributes":{},"id":"357337","type":"BasicTicker"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"357365","type":"BasicTickFormatter"},"plot":{"id":"357322","subtype":"Figure","type":"Plot"},"ticker":{"id":"357337","type":"BasicTicker"}},"id":"357336","type":"LinearAxis"},{"attributes":{"data_source":{"id":"357356","type":"ColumnDataSource"},"glyph":{"id":"357357","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"357358","type":"VBar"},"selection_glyph":null,"view":{"id":"357360","type":"CDSView"}},"id":"357359","type":"GlyphRenderer"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.09},"x":{"field":"x"}},"id":"357358","type":"VBar"},{"attributes":{"below":[{"id":"357332","type":"CategoricalAxis"}],"left":[{"id":"357336","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"357332","type":"CategoricalAxis"},{"id":"357335","type":"Grid"},{"id":"357336","type":"LinearAxis"},{"id":"357340","type":"Grid"},{"id":"357349","type":"BoxAnnotation"},{"id":"357359","type":"GlyphRenderer"}],"title":{"id":"357321","type":"Title"},"toolbar":{"id":"357347","type":"Toolbar"},"x_range":{"id":"357324","type":"FactorRange"},"x_scale":{"id":"357328","type":"CategoricalScale"},"y_range":{"id":"357326","type":"DataRange1d"},"y_scale":{"id":"357330","type":"LinearScale"}},"id":"357322","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Drug Counts"},"id":"357321","type":"Title"},{"attributes":{"callback":null,"start":0},"id":"357326","type":"DataRange1d"},{"attributes":{"callback":null,"factors":["strain","joints","fish","fluoxetine"],"range_padding":0.01},"id":"357324","type":"FactorRange"},{"attributes":{},"id":"357328","type":"CategoricalScale"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"357349","type":"BoxAnnotation"},{"attributes":{},"id":"357330","type":"LinearScale"},{"attributes":{"dimension":1,"plot":{"id":"357322","subtype":"Figure","type":"Plot"},"ticker":{"id":"357337","type":"BasicTicker"}},"id":"357340","type":"Grid"},{"attributes":{},"id":"357341","type":"PanTool"},{"attributes":{},"id":"357342","type":"WheelZoomTool"},{"attributes":{"overlay":{"id":"357349","type":"BoxAnnotation"}},"id":"357343","type":"BoxZoomTool"},{"attributes":{},"id":"357344","type":"SaveTool"},{"attributes":{},"id":"357363","type":"CategoricalTickFormatter"},{"attributes":{},"id":"357346","type":"HelpTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"357341","type":"PanTool"},{"id":"357342","type":"WheelZoomTool"},{"id":"357343","type":"BoxZoomTool"},{"id":"357344","type":"SaveTool"},{"id":"357345","type":"ResetTool"},{"id":"357346","type":"HelpTool"}]},"id":"357347","type":"Toolbar"},{"attributes":{},"id":"357366","type":"Selection"},{"attributes":{},"id":"357367","type":"UnionRenderers"}],"root_ids":["357322"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('357425').textContent;
                  var render_items = [{"docid":"5b47b444-ddae-40e3-9b26-3532ef1dc2a5","roots":{"357322":"be2c1604-b3df-4edf-aee0-755e3ccb2788"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>