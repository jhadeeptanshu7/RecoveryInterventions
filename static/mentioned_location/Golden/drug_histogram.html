



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="53a3931b-3d6a-4e0a-9908-751a2cc29400"></div>
            
          
        
      
      
        <script type="application/json" id="174607">
          {"c7ffdfff-1639-4f89-a237-a01c74f7ab7f":{"roots":{"references":[{"attributes":{"axis_label":"Drug","formatter":{"id":"174545","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"174504","subtype":"Figure","type":"Plot"},"ticker":{"id":"174515","type":"CategoricalTicker"}},"id":"174514","type":"CategoricalAxis"},{"attributes":{},"id":"174515","type":"CategoricalTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"174504","subtype":"Figure","type":"Plot"},"ticker":{"id":"174515","type":"CategoricalTicker"}},"id":"174517","type":"Grid"},{"attributes":{},"id":"174510","type":"CategoricalScale"},{"attributes":{},"id":"174512","type":"LinearScale"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"174547","type":"BasicTickFormatter"},"plot":{"id":"174504","subtype":"Figure","type":"Plot"},"ticker":{"id":"174519","type":"BasicTicker"}},"id":"174518","type":"LinearAxis"},{"attributes":{},"id":"174519","type":"BasicTicker"},{"attributes":{"callback":null,"data":{"top":[6,5,5,5,5,4,4,4,4,4,4,4,4,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["speed","black","sub","glass","e","crack","purple","hits","bars","white","north","oil","lines","nicotine","fish","k","subs","pot","pg","fire","dmt","cigs","weed","cigarettes","lsd","ketamine","tobacco","h","l","crown","acid","ice","cubensis","blow","hash","heroin","cheese","banana","psilocybin","bud","spice","j","ms","marijuana","strain","dxm","meth","blunt","xanax","nbome","gt","tincture","benzo","shrooms","liberty","zinc","dopamine","alp","cola","citalopram","trees","tbd","pst","benadryl","dextromethorphan","concerta","dabs","gabapentin","klonopin","bowls","kratom","edibles","crystal","cannabis","haze","420","dck","polistirex","nmda","changa","mdma","benzos","anxiolytics","salts","xr","cocaine","bongs","buds","cbd","tesla","m1","25i","cope","coke","serotonin","md","calcium","tar","nexus","ssri","seed","crystals","melatonin","avb","bowl","snow","vicodin","80s","lorazepam","ibogaine","mph","morphine","doc","acids"]},"selected":{"id":"174548","type":"Selection"},"selection_policy":{"id":"174549","type":"UnionRenderers"}},"id":"174538","type":"ColumnDataSource"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"174539","type":"VBar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"174531","type":"BoxAnnotation"},{"attributes":{},"id":"174548","type":"Selection"},{"attributes":{},"id":"174545","type":"CategoricalTickFormatter"},{"attributes":{"plot":null,"text":"Drug Counts"},"id":"174503","type":"Title"},{"attributes":{"callback":null,"factors":["speed","black","sub","glass","e","crack","purple","hits","bars","white","north","oil","lines","nicotine","fish","k","subs","pot","pg","fire","dmt","cigs","weed","cigarettes","lsd","ketamine","tobacco","h","l","crown","acid","ice","cubensis","blow","hash","heroin","cheese","banana","psilocybin","bud","spice","j","ms","marijuana","strain","dxm","meth","blunt","xanax","nbome","gt","tincture","benzo","shrooms","liberty","zinc","dopamine","alp","cola","citalopram","trees","tbd","pst","benadryl","dextromethorphan","concerta","dabs","gabapentin","klonopin","bowls","kratom","edibles","crystal","cannabis","haze","420","dck","polistirex","nmda","changa","mdma","benzos","anxiolytics","salts","xr","cocaine","bongs","buds","cbd","tesla","m1","25i","cope","coke","serotonin","md","calcium","tar","nexus","ssri","seed","crystals","melatonin","avb","bowl","snow","vicodin","80s","lorazepam","ibogaine","mph","morphine","doc","acids"],"range_padding":0.01},"id":"174506","type":"FactorRange"},{"attributes":{"below":[{"id":"174514","type":"CategoricalAxis"}],"left":[{"id":"174518","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"174514","type":"CategoricalAxis"},{"id":"174517","type":"Grid"},{"id":"174518","type":"LinearAxis"},{"id":"174522","type":"Grid"},{"id":"174531","type":"BoxAnnotation"},{"id":"174541","type":"GlyphRenderer"}],"title":{"id":"174503","type":"Title"},"toolbar":{"id":"174529","type":"Toolbar"},"x_range":{"id":"174506","type":"FactorRange"},"x_scale":{"id":"174510","type":"CategoricalScale"},"y_range":{"id":"174508","type":"DataRange1d"},"y_scale":{"id":"174512","type":"LinearScale"}},"id":"174504","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null,"start":0},"id":"174508","type":"DataRange1d"},{"attributes":{"source":{"id":"174538","type":"ColumnDataSource"}},"id":"174542","type":"CDSView"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"174523","type":"PanTool"},{"id":"174524","type":"WheelZoomTool"},{"id":"174525","type":"BoxZoomTool"},{"id":"174526","type":"SaveTool"},{"id":"174527","type":"ResetTool"},{"id":"174528","type":"HelpTool"}]},"id":"174529","type":"Toolbar"},{"attributes":{},"id":"174528","type":"HelpTool"},{"attributes":{"data_source":{"id":"174538","type":"ColumnDataSource"},"glyph":{"id":"174539","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"174540","type":"VBar"},"selection_glyph":null,"view":{"id":"174542","type":"CDSView"}},"id":"174541","type":"GlyphRenderer"},{"attributes":{},"id":"174549","type":"UnionRenderers"},{"attributes":{},"id":"174526","type":"SaveTool"},{"attributes":{},"id":"174547","type":"BasicTickFormatter"},{"attributes":{},"id":"174523","type":"PanTool"},{"attributes":{"dimension":1,"plot":{"id":"174504","subtype":"Figure","type":"Plot"},"ticker":{"id":"174519","type":"BasicTicker"}},"id":"174522","type":"Grid"},{"attributes":{"overlay":{"id":"174531","type":"BoxAnnotation"}},"id":"174525","type":"BoxZoomTool"},{"attributes":{},"id":"174524","type":"WheelZoomTool"},{"attributes":{},"id":"174527","type":"ResetTool"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"174540","type":"VBar"}],"root_ids":["174504"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('174607').textContent;
                  var render_items = [{"docid":"c7ffdfff-1639-4f89-a237-a01c74f7ab7f","roots":{"174504":"53a3931b-3d6a-4e0a-9908-751a2cc29400"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>